### PR TITLE
Adding info for IsDisplayed() selenium method

### PR DESCRIPTION
**Pull request by MartinoHead**
The information about IsDisplayed() method is missing. Examples are also missing. I am adding basic description and later will add examples.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/elements/information.en.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ There are a number of details you can query about a specific element.
 
 ## Is Displayed
 
+The isDisplayed method in Selenium verifies if a certain element is present and displayed. If the element is displayed, then the value returned is true. If not, then the value returned is false.
```
Comment (diemol) [inline website_and_docs/content/documentation/webdriver/elements/information.en.md]: Please use max 100 chars per line.

**Review by diemol — CHANGES_REQUESTED** [review]
Could you please add this information to the other `information.*.md` files?

**Comment by MartinoHead** [comment]
> Could you please add this information to the other `information.*.md` files?

I don't have this description in other languages. Should i just use google traslator?

**Comment by diemol** [comment]
> > Could you please add this information to the other `information.*.md` files?
> 
> I don't have this description in other languages. Should i just use google traslator?

There is no need to translate, the English content is fine. One of the translators will have a look at it.

**Comment by harsha509** [comment]
Hi @MartinoHead,

Thank you for taking time and raising the PR.

I was not aware of this PR but and replaced this PR with commit https://github.com/SeleniumHQ/seleniumhq.github.io/commit/9cfe79bcda83e4c7d84c884b4b5a990c8159a4c3 !

Thanks,
Harsha

**Pull request closed without merging**